src/pages/calendarPage.js

```diff
@@ -1,14 +1,27 @@
-import React from 'react';
+import React, { Component } from 'react';
 import StickyNavBar from '../components/stickyNavBar';
 import '../styles/calendarPage.css';
-
-function CalendarPage() {
+class CalendarPage extends Component {
```

Comment:
Implementing components as classes is no longer recommended, from [Component Legacy React API](https://react.dev/reference/react/Component): 

> Class components are still supported by React, but we don’t recommend using them in new code. 

Let's keep this calendar page as a function declared component and nest a google calendar component into it.